### PR TITLE
[WIP] add instructions to README.md

### PR DESCRIPTION
**Pull request by thatfiredev**

### CODE REVIEW

**Comment by jamesdaniels** [comment]
Thanks for hacking on this, FYI the tests are a little flaky right now. Namely Node 14 on macos, so ignore, I disabled the branch protection policy for now.

File: README.md
Comment (davideast) [inline README.md:78]: Switch with Angular, Nuxt is still a WIP for now.

```suggestion
# Deploy Angular
```

File: README.md
Comment (davideast) [inline README.md:32]:
```suggestion
# Deploy Next.js
```

**Review by davideast — APPROVED** [review]
LGTM with the minor changes.

File: README.md
Comment (davideast) [inline README.md:85]: `10.9.1` is required now.

Comment (davideast) [inline README.md:85]:
```suggestion
- Firebase CLI version 10.9.1 or later (see installation instructions [here](https://firebase.google.com/docs/cli))
```

File: README.md
Comment (davideast) [inline README.md:39]:
```suggestion
- Firebase CLI version 10.9.1 or later (see installation instructions [here](https://firebase.google.com/docs/cli))
```

**Comment by thatfiredev** [comment]
@jamesdaniels this still requires a maintainer to approve before it can be merged. Can you take a look and merge?

**Comment by morganchen12** [comment]
Skipping tests and review and merging since it's only README changes per David's request.

**Pull request merged**
